airflow/contrib/operators/cloudml_operator.py

```diff
@@ -106,6 +116,199 @@ def _normalize_cloudml_job_id(job_id):
     return re.sub('[^0-9a-zA-Z]+', '_', job_id)
 
 
+def create_evaluate_ops(task_prefix,
```

Comment:
This seems like a static util method. I'm personally a bit nervous about embedding this in the CloudML operator. What do you think about moving this to a separate cloudml_operator_utils.py class?